modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/common/services/liferay/rest/raysource/LicenseKeys.js

```diff
@@ -90,3 +90,22 @@ export async function getActivationDownloadKey(
 
 	return response;
 }
+
+export async function getAllActivationKeys(
```

Comment:
Analyzing the endpoint, I guess that we should use this name for function:

```suggestion
export async function exportLicenseKeys(
```

Reply:
I agreed, it's done! Thank you!